christin/index.js

```diff
@@ -0,0 +1,174 @@
+//1.Reverse
+// Create a function to reverse a number.
+//Examples:reverse(34532) -> expected output: 23543
+
+function numberReverse() {
+  let numbers = 84337;
+  let array = numbers.toString().split("");
+  array.reverse();
+  let joinArray = parseInt(array.join(""));
+  return joinArray;
+}
+console.log(numberReverse());
+
+// 2. Alphabetical Order
+//Create a function to sort a string into alphabetical order.Examples: <<<<<< < HEAD
+
+//alphaOrder("webdev") -> expected output: "bdeevw"
+
+//Note: Assume numbers, symbols and punctuation are not included in the string.
+
+function getAlphabeticalOrder() {
+  let string = "development";
+  let sortString = string.split("");
+  sortString.sort();
+  let joinString = sortString.join("");
+  return joinString;
+}
+
+console.log(getAlphabeticalOrder());
+
+//? 3. The Greater Numbers
+//?Create a function which accepts two arguments, an array of numbers and a number.The function should return the elements of the array which are greater than the second argument.Examples:*/
+
+//findGreatest([3, 4, 5], 4) -> expected output: 5
+//findGreatest([10, 20, 30], 12) -> expected output: 20, 30
+//findGreatest([0, 10, 3], 4) -> expected output: 10
+
+let box = [];
+
+function getGreatestNumber(numb2, startNum) {
+  for (let i = 0; i > 3; i++) {
+    if (numb2[i] > startNum) {
+      box.push(numb2[i]);
+    }
+  }
+  return box;
+}
+
```

Comment:
I still get an empty array when I try to run the code!

Reply:
solved, please check all again